src/main/java/edu/ucsb/cs56/ucsb_open_lab_scheduler/controllers/InstructorMenuController.java

```diff
@@ -89,4 +98,20 @@ public String getTutor(@PathVariable("id") long id, Model model, OAuth2Authentic
         return "instructorMenu/getTutors";
     }
 
+    @GetMapping("/InstructorMenu/downloadCSV/{id}")
```

Comment:
Can you return a view here. This endpoint is not protected. We only want people to access this endpoint if they are instructors. Take a look at how li-el protected room availability csv.

Reply:
Hi, I just fixed this problem, could you take a look if there is further problem? Thanks.